<!--
DESCRITPION
-->


<launch>
        <!--node pkg="ira_laser_tools" name="camerascan_multi_merger" type="laserscan_multi_merger" output="screen">
        	<param name="destination_frame" value="base_link"/>
		<param name="cloud_destination_topic" value="/merged_cloud"/>
		<param name="scan_destination_topic" value="/camera/scan"/>
                <param name="laserscan_topics" value="/camera_left/scan /camera_right/scan" />

                <param name="angle_min" value="-3.0"/>
                <param name="angle_max" value="3.0"/>
                <param name="angle_increment" value="0.0058"/>
                <param name="scan_time" value="0.0333333"/>
                <param name="range_min" value="0.05"/>
                <param name="range_max" value="15.0"/>

	</node>
        <node pkg="ira_laser_tools" name="laserscan_multi_merger" type="laserscan_multi_merger" output="screen">
                <param name="destination_frame" value="base_link"/>
                <param name="cloud_destination_topic" value="/merged_cloud1"/>
                <param name="scan_destination_topic" value="/laser/scan"/>
                <param name="laserscan_topics" value="/laser_left/scan /laser_right/scan"/>

                <param name="angle_min" value="-3.0"/>
                <param name="angle_max" value="3.0"/>
                <param name="angle_increment" value="0.0058"/>
                <param name="scan_time" value="0.0333333"/>
                <param name="range_min" value="0.05"/>
                <param name="range_max" value="15.0"/>

        </node-->
        <node pkg="ira_laser_tools" name="cameralaser_multi_merger" type="laserscan_multi_merger" output="screen">
                <param name="destination_frame" value="base_link"/>
                <param name="cloud_destination_topic" value="/merged_cloud"/>
                <param name="scan_destination_topic" value="/scan"/>
                <param name="laserscan_topics" value="/camera_right/scan /camera_left/scan /laser_right/scan /laser_left/scan"/>

                <param name="angle_min" value="-3.0"/>
                <param name="angle_max" value="3.0"/>
                <param name="angle_increment" value="0.0058"/>
                <param name="scan_time" value="0.0333333"/>
                <param name="range_min" value="0.05"/>
                <param name="range_max" value="15.0"/>

        </node>
        <!--node pkg="ira_laser_tools" name="cameralaser_multi_merger" type="laserscan_multi_merger" output="screen">
                <param name="destination_frame" value="base_link"/>
                <param name="cloud_destination_topic" value="/merged_cloud2"/>
                <param name="scan_destination_topic" value="/scan"/>
                <param name="laserscan_topics" value="/laser/scan /camera/scan"/>

                <param name="angle_min" value="-3.0"/>
                <param name="angle_max" value="3.0"/>
                <param name="angle_increment" value="0.0058"/>
                <param name="scan_time" value="0.0333333"/>
                <param name="range_min" value="0.05"/>
                <param name="range_max" value="15.0"/>

        </node-->

</launch>
